packs/equipment-effects/effect-beastmasters-sigil-ranged-weapon-greater.json

```diff
@@ -1,6 +1,6 @@
 {
     "_id": "TU67AK08CUsP7pl4",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/creatures/mammals/bat-giant-tattered-purple.webp",
```

Comment:
Did you intend for the greater, major, and normal to have different icons?

Reply:
Different animals per the description, so I was going for that.